.pre-commit-config.yaml

```diff
@@ -31,4 +31,4 @@ repos:
         name: Don't commit common R artifacts
         entry: Cannot commit .Rhistory, .RData, .Rds or .rds.
         language: fail
-        files: '\.(Rhistory|RData|Rds)$'
+        files: '\.(Rhistory|RData|Rds|rds)$'
```

Comment:
No need to skip this pre-commit check if there are no `.rds` files in your local repo.